src/main/java/core/basesyntax/SqrtOperation.java

```diff
@@ -7,7 +7,10 @@
  * где вместо вопросика будет подставлено фактически переданное в метод число.</p>
  */
 public class SqrtOperation {
-    public double sqrt(double number) {
-        return 0.0;
+    public double sqrt(double number) throws IllegalArgumentException {
```

Comment:
do we need exception in method signature?

Reply:
don`t sure.
Removed

Reply:
it's better not just to remove it, but find out why, so i suggest you google it, and if you don't figure out on your own let's visit QnA and ask mentor about it, deal?

Reply:
Okay, as i see, it`s supposed to escalate error to deal with in programs, which would use it further.